platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ImportMigrationContext.java

```diff
@@ -131,4 +132,15 @@
    * </code>
    */
   public Stream<ImportMigrationEntry> entries(Path path, PathMatcher filter);
+
+  /**
+   * Retrieves exported system property.
+   *
+   * @param key the key of the system property to retrieve
+   * @return the system property requested
```

Comment:
✏️ `the system property requested or null if the system property was not exported`